aten/src/ATen/core/TensorImpl.h

```diff
@@ -264,12 +264,12 @@ struct CAFFE2_API TensorImpl : public c10::intrusive_ptr_target {
   template <typename T>
   inline T * data() const {
     AT_ASSERT(!is_variable());
-    CAFFE_ENFORCE_WITH_CALLER(
+    AT_ENFORCE_WITH_CALLER(
```

Comment:
cc'ing @salexspb for a quick review. This actually removes the functionality of printing out the caller. I am fine with it but we should be explicit. See comment above.

Reply:
Does it? Should be exactly like the caffe version.

Reply:
@Yangqing , enforce with a caller was a pretty useful feature for debugging.  I personally haven't relied on it recently but don't know how many people are.